webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyScreenshot.java

```diff
@@ -23,7 +23,7 @@ public class ThirdPartyScreenshot extends AuditableEntity {
   @Column(name = "third_party_id")
   String thirdPartyId;
 
-  @OneToOne(fetch = FetchType.LAZY)
+  @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Why is this change required? It looks like a fundamental change in the entity relationship and not related to a library uptick

Reply:
I made the change to fix a warning message at compile time, but I reverted it.